### PR TITLE
[Keyboard] Add aliceh66 pianoforte

### PR DESCRIPTION
**Pull request by AliceH66**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/aliceh66/pianoforte/info.json [hunk 1]
```diff
@@ -0,0 +1,497 @@
+{
+    "keyboard_name": "pianoforte solder PCB",
+    "url": "https://github.com/AliceH66",
+    "maintainer": "AliceH66",
```
Comment (drashna) [inline keyboards/aliceh66/pianoforte/info.json:4]:
```suggestion
    "maintainer": "AliceH66",
    "manufacturer": AlliceH",
    "usb": {
        "device_version": "1.0.0",
        "pid": "0x7066",
        "vid": "0x6168"
    },
```

File: keyboards/aliceh66/pianoforte/keymaps/via/rules.mk [hunk 2]
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
```
Comment (drashna) [inline keyboards/aliceh66/pianoforte/keymaps/via/rules.mk:2]: https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs
> Vial-related files or changes will not be accepted, as they are not used by QMK firmware (no Vial-specific core code has been submitted or merged)

```suggestion
```

File: keyboards/aliceh66/pianoforte/keymaps/vial/config.h [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```
Comment (drashna) [inline keyboards/aliceh66/pianoforte/keymaps/vial/config.h:1]: https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs
> Vial-related files or changes will not be accepted, as they are not used by QMK firmware (no Vial-specific core code has been submitted or merged)

File: keyboards/aliceh66/pianoforte/keymaps/vial/keymap.c [hunk 4]
```diff
@@ -0,0 +1,33 @@
+/*
```
Comment (drashna) [inline keyboards/aliceh66/pianoforte/keymaps/vial/keymap.c:1]: https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs
> Vial-related files or changes will not be accepted, as they are not used by QMK firmware (no Vial-specific core code has been submitted or merged)

File: keyboards/aliceh66/pianoforte/keymaps/vial/pianoforte/pianoforte.json [hunk 5]
```diff
@@ -0,0 +1,294 @@
+{
```
Comment (drashna) [inline keyboards/aliceh66/pianoforte/keymaps/vial/pianoforte/pianoforte.json:1]: https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs
> Vial-related files or changes will not be accepted, as they are not used by QMK firmware (no Vial-specific core code has been submitted or merged)

File: keyboards/aliceh66/pianoforte/keymaps/vial/rules.mk [hunk 6]
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
```
Comment (drashna) [inline keyboards/aliceh66/pianoforte/keymaps/vial/rules.mk:1]: https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs
> Vial-related files or changes will not be accepted, as they are not used by QMK firmware (no Vial-specific core code has been submitted or merged)

File: keyboards/aliceh66/pianoforte_hs/config.h [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+*/
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x6168
+#define PRODUCT_ID      0x7068
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    AliceH 
+#define PRODUCT         pianoforte_hotswap
```
Comment (drashna) [inline keyboards/aliceh66/pianoforte_hs/config.h:25]:
```suggestion
```

File: keyboards/aliceh66/pianoforte_hs/info.json [hunk 8]
```diff
@@ -0,0 +1,104 @@
+{
+    "keyboard_name": "pianoforte hotswap PCB",
+    "url": "https://github.com/AliceH66",
+    "maintainer": "AliceH66",
```
Comment (drashna) [inline keyboards/aliceh66/pianoforte_hs/info.json:4]:
```suggestion
    "maintainer": "AliceH66",
    "manufacturer": AliceH",
    "usb": {
        "device_version": "1.0.0",
        "pid": "0x7068",
        "vid": "0x6168"
    },
```

File: keyboards/aliceh66/pianoforte_hs/keymaps/via/rules.mk [hunk 9]
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
```
Comment (drashna) [inline keyboards/aliceh66/pianoforte_hs/keymaps/via/rules.mk:2]:
```suggestion
```

File: keyboards/aliceh66/pianoforte_hs/keymaps/vial/config.h [hunk 10]
```diff
@@ -0,0 +1,7 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```
Comment (drashna) [inline keyboards/aliceh66/pianoforte_hs/keymaps/vial/config.h:1]: same

File: keyboards/aliceh66/pianoforte_hs/keymaps/vial/pianoforte_hs/pianoforte_hs.json [hunk 11]
```diff
@@ -0,0 +1,214 @@
+{
```
Comment (drashna) [inline keyboards/aliceh66/pianoforte_hs/keymaps/vial/pianoforte_hs/pianoforte_hs.json:1]: same

File: keyboards/aliceh66/pianoforte_hs/rules.mk [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+QMK_SETTINGS = no 			# QMK Settings
```
Comment (drashna) [inline keyboards/aliceh66/pianoforte_hs/rules.mk:23]:
```suggestion
```

File: keyboards/aliceh66/pianoforte/rules.mk [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+QMK_SETTINGS = no			# QMK Settings
```
Comment (drashna) [inline keyboards/aliceh66/pianoforte/rules.mk:23]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by AliceH66** [comment]
I committed to the suggestions and removed the VIAL files, I believe that should be it?

File: keyboards/aliceh66/pianoforte/config.h [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+#define DIODE_DIRECTION COL2ROW
+
+/* define if matrix has ghost */
+//#define MATRIX_HAS_GHOST
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE    5
+#define VIAL_KEYBOARD_UID {0x37, 0xFC, 0xE4, 0xB0, 0x9B, 0xC6, 0x1A, 0xD8}
+#define VIAL_UNLOCK_COMBO_ROWS { 0, 7 }
+#define VIAL_UNLOCK_COMBO_COLS { 0, 6 }
```
Comment (drashna) [inline keyboards/aliceh66/pianoforte/config.h:35]:
```suggestion
```

File: keyboards/aliceh66/pianoforte_hs/config.h [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+#define DIODE_DIRECTION COL2ROW
+
+/* define if matrix has ghost */
+//#define MATRIX_HAS_GHOST
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE    5
+#define VIAL_KEYBOARD_UID {0xF3, 0xC1, 0x73, 0xB5, 0x52, 0xD4, 0x0E, 0x99}
+#define VIAL_UNLOCK_COMBO_ROWS { 0, 6 }
+#define VIAL_UNLOCK_COMBO_COLS { 0, 6 }
```
Comment (drashna) [inline keyboards/aliceh66/pianoforte_hs/config.h:35]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/aliceh66/pianoforte_hs/info.json [hunk 16]
```diff
@@ -0,0 +1,110 @@
...
+    "manufacturer": AliceH",
```
Comment (fauxpark) [inline keyboards/aliceh66/pianoforte_hs/info.json:5]:
```suggestion
    "keyboard_name": "pianoforte hotswap",
    "manufacturer": "AliceH",
    "url": "https://github.com/AliceH66",
    "maintainer": "AliceH66",
```

File: keyboards/aliceh66/pianoforte/info.json [hunk 17]
```diff
@@ -0,0 +1,503 @@
...
+    "manufacturer": AlliceH",
```
Comment (fauxpark) [inline keyboards/aliceh66/pianoforte/info.json:5]:
```suggestion
    "keyboard_name": "pianoforte solder",
    "manufacturer": "AliceH",
    "url": "https://github.com/AliceH66",
    "maintainer": "AliceH66",
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/aliceh66/pianoforte/readme.md [hunk 18]
```diff
@@ -0,0 +1,14 @@
+# AliceH pianoforte solder PCB 
+
+QMK Firmware for the pianoforte solder PCB
+
+* Keyboard Maintainer: [AliceH] (https://github.com/AliceH66)
+* Hardware Supported: pianoforte Solder
+
+Make example for this keyboard (after setting up your build environment):
+	make aliceh66/pianoforte:default
```
Comment (fauxpark) [inline keyboards/aliceh66/pianoforte/readme.md:9]:
```suggestion

    make aliceh66/pianoforte:default
```

File: keyboards/aliceh66/pianoforte/rules.mk [hunk 19]
```diff
@@ -0,0 +1,24 @@
...
+LTO_ENABLE = yes			# Enable link time optomization
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
```
Comment (fauxpark) [inline keyboards/aliceh66/pianoforte/rules.mk:22]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
LTO_ENABLE = yes
```

File: keyboards/aliceh66/pianoforte_hs/readme.md [hunk 20]
```diff
@@ -0,0 +1,14 @@
+# AliceH pianoforte hotswap PCB 
+
+QMK Firmware for the pianoforte hotswap PCB
+
+* Keyboard Maintainer: [AliceH] (https://github.com/AliceH66)
+* Hardware Supported: pianoforte hotswap
+
+Make example for this keyboard (after setting up your build environment):
+	make aliceh66/pianoforte_hs:default
```
Comment (fauxpark) [inline keyboards/aliceh66/pianoforte_hs/readme.md:9]:
```suggestion

    make aliceh66/pianoforte_hs:default
```

File: keyboards/aliceh66/pianoforte_hs/rules.mk [hunk 21]
```diff
@@ -0,0 +1,22 @@
...
+LTO_ENABLE = yes			# Enable link time optomization
+BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
```
Comment (fauxpark) [inline keyboards/aliceh66/pianoforte_hs/rules.mk:22]:
```suggestion
BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
LTO_ENABLE = yes
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**